### PR TITLE
Fix typos in kernel.cr comments

### PR DESCRIPTION
**Pull request by pgkos**
This fixes some typos in `kernel.cr` documentation comments.

### CODE REVIEW

File: src/kernel.cr [hunk 1]
```diff
@@ -27,24 +27,24 @@ def gets(*args)
   STDIN.gets(*args)
 end
 
-# Reads a line from STDIN. See `IO#read_line`.
+# Reads a line from STDOUT. See `IO#read_line`.
```
Comment (chdorner) [inline src/kernel.cr]: Isn't the code actually reading from `STDIN` here?

Comment (jhass) [inline src/kernel.cr]: Oh but actually that's not true. stdout is where things are written to (in shell what gets redirected for `a` in `a | b` and `a > ./file`), stdin is where things are read from (in shell what gets redirected for `b` in `a | b` and `b < ./file`).

File: src/kernel.cr [hunk 2]
```diff
@@ -27,24 +27,24 @@ def gets(*args)
...
 def read_line(*args)
   STDIN.read_line(*args)
 end
 
-# Prints objects to STDIN. See `IO#print`.
+# Prints objects to STDOUT. See `IO#print`.
```
Comment (jhass) [inline src/kernel.cr]: This correction is good though :)

**Comment by pgkos** [comment]
sorry :smile:

I have amended the commit

**Comment by jhass** [comment]
Thank you!

**Pull request merged**